scipy/stats/_morestats.py

```diff
@@ -4183,3 +4184,166 @@ def directional_stats(samples, *, axis=0, normalize=True):
     mean_direction = mean / mean_resultant_length
     return DirectionalStats(mean_direction,
                             mean_resultant_length.squeeze(-1)[()])
+
+
+def false_discovery_control(ps, *, axis=0, method='bh'):
+    """Adjust p-values to control the false discovery rate (FDR).
+
+    If the null hypothesis is rejected when the *adjusted* p-value falls below
+    a specified level, the false discovery rate is controlled at that level.
```

Comment:
This might be a bit hard to understand if you don't know anything about FDR

Reply:
```suggestion
    If the null hypothesis is rejected when the *adjusted* p-value falls below
    a specified level, the false discovery rate - that is, the fraction of
    rejected null hypotheses that were actually true - is controlled at that
    level.
```
Would this help?
That said, I'd encourage the user to know something about stats functions before using them.

Reply:
I know; we've discussed this quite a bit before. Our opinions are a little different, so we might not always agree on these points. That's OK.